content/400-pulse/200-getting-started.mdx

```diff
@@ -113,6 +116,10 @@ SELECT pg_reload_conf();
 
 </details>
 
+##### ssl mode
```

Comment:
either uppercase for proper spelling or write in backticks / code formatting and one word (`sslmode`) to indicate you're refering to the setting in code.

```suggestion
##### SSL mode
```

Reply:
Kept the upper case notation as a suggestion!